config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
+  get 'arts/index'
+  get 'arts/new'
+  get 'arts/create'
   root to: "pages#home"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # resources for Art model
+  resources :arts, only: %i[index new create]
```

Comment:
The show is missing.
You'll also need to add it to the controller and make a view for it like you did for the new and index

Reply:
added it now